### PR TITLE
Space Gorillas

### PR DESCRIPTION
**Pull request by jewishneocon**
ayy
hell yeah motherfucker we now got space gorillas in the game ooh ooh ah ah banana

### CODE REVIEW

**Comment by Zuhayr** [comment]
I know I'm not a contributor to /vg/ but this is legitimately terrible.

**Comment by Iamgoofball** [comment]
0/10 not a revolver overhaul it's OK - IGN

**Comment by BestName** [comment]
Gorillionaires and bootyborgs when?

**Comment by jewishneocon** [comment]
how is it terrible baybie

Gorillionaires now.

File: code/modules/mob/living/simple_animal/friendly/gorilla.dm [hunk 1]
```diff
@@ -0,0 +1,25 @@
+//oo oo ah ah
+/mob/living/simple_animal/gorilla
+	name = "Space Gorilla"
+	desc = "GRAPE APE GRAPE APE"
```
Comment (Zuhayr) [inline code/modules/mob/living/simple_animal/friendly/gorilla.dm]: No fullstop.

File: code/modules/mob/living/simple_animal/friendly/gorilla.dm [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	icon_state = "spacegoriilla"
+	icon_living = "spacegorilla"
+	icon_dead = "deadgorilla"
+	gender = MALE
+	speak = list("OOH OOH AH AH!", "BANANA",)
+	speak_emote = list("chimpers")
+	emote_hear = list("chimpers", "grunts")
+	emote_see = list("pounds his chest", "grunts OOH OOH AH AH")
```
Comment (Zuhayr) [inline code/modules/mob/living/simple_animal/friendly/gorilla.dm]: No speech formatting.

File: code/modules/mob/living/simple_animal/friendly/gorilla.dm [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+	speak_chance = 1
+	turns_per_move = 5
+	see_in_dark = 6
+	species = /mob/living/simple_animal/gorilla
+	meat_type = /obj/item/weapon/reagent_containers/food/snacks/meat
+	response_help  = "pas"
```
Comment (Zuhayr) [inline code/modules/mob/living/simple_animal/friendly/gorilla.dm]: Pas.

File: code/modules/mob/living/simple_animal/friendly/gorilla.dm [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+	response_disarm = "gently pushes aside"
+	response_harm   = "kicks"
+	min_oxy = 16      // Require atleast 16kPA oxygen
```
Comment (Zuhayr) [inline code/modules/mob/living/simple_animal/friendly/gorilla.dm]: Pointless commenting.

File: code/modules/mob/living/simple_animal/friendly/gorilla.dm [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+	minbodytemp = 223 // Below -50 Degrees Celcius
+	maxbodytemp = 323 // Above 50 Degrees Celcius
+	var/turns_since_scan = 0
```
Comment (Zuhayr) [inline code/modules/mob/living/simple_animal/friendly/gorilla.dm]: Did you actually port the procs that use these vars or are you copypaste defining them for no reason?

File: code/modules/mob/living/simple_animal/friendly/gorilla.dm [hunk 6]
```diff
@@ -0,0 +1,25 @@
+//oo oo ah ah
+/mob/living/simple_animal/gorilla
+	name = "Space Gorilla"
```
Comment (Zuhayr) [inline code/modules/mob/living/simple_animal/friendly/gorilla.dm]: Proper noun capitalization will break \the.

**Comment by Zuhayr** [comment]
Reviewed, since you asked.

File: code/modules/mob/living/simple_animal/friendly/gorilla.dm (hunk 5 above)
Comment (jewishneocon) [inline code/modules/mob/living/simple_animal/friendly/gorilla.dm]: i copypasta'd cat code

**Comment by jewishneocon** [comment]
thanks man

**Comment by jewishneocon** [comment]
closed for shit and ill fix erros

**Pull request closed without merging**